src/modules/holder/pages/Home/Home.tsx

```diff
@@ -31,7 +35,12 @@ export const Home: FC = () => {
     )
   }
 
-  if (data.length === 0) {
+  const tickets = data.filter((credentialItem) => {
+    const credentialSchema = (credentialItem as StoredW3CCredential).credentialSchema
+    return credentialSchema?.id === 'https://schema.affinidi.com/EventEligibilityV1-0.json'
```

Comment:
move this value to root `utils` folder and also update it for `src/modules/issuer/CredentialForm/useCredentialForm.ts` file